drivers/flash/nrf_qspi_nor.c

```diff
@@ -189,7 +189,7 @@ static inline nrf_qspi_frequency_t get_nrf_qspi_prescaler(uint32_t frequency)
 	if (frequency < 2000000UL) {
 		ret = -EINVAL;
 	} else if (frequency >= 32000000UL) {
-		ret = NRF_QSPI_FREQ_32MDIV1;
+		ret = NRF_QSPI_FREQ_DIV1;
```

Comment:
Is this change necessary?  AFAICT 

    #define NRF_QSPI_FREQ_32MDIV1  NRF_QSPI_FREQ_DIV1

in the HAL.

Reply:
This is only defined for NRF52_SERIES, not for nRF53. I understood this as a deprecated alias.

Reply:
Looking at the clock tree on the nRF53, my change is likely wrong. It will compile on both nRF52 and nRF53, but give wrong dividers on the latter.

It seems that on nRF53, the QSPI is clocked from HFCLK192M which is by default divided by 4, so 48MHz.

get_nrf_qspi_prescaler would need to be adapted specifically for nRF53.

Reply:
OK.  Then please take that out, so we can at least fix the opcode issue.  We can address nRF53 support in 2.5.0.